webview-ui/src/kilocode/agent-manager/state/atoms/sessions.ts

```diff
@@ -65,6 +121,10 @@ export const removeSessionAtom = atom(null, (get, set, sessionId: string) => {
 	}
 })
 
+export const setPendingSessionAtom = atom(null, (_get, set, pending: PendingSession | null) => {
```

Comment:
These atoms which just set another atom in a non-complicated way seem redundant? (here and L162)